src/redux/migrations.ts

```diff
@@ -230,4 +230,13 @@ export default {
       wordByWordFontScale: initialState.quranReaderStyles.wordByWordFontScale,
     },
   }),
+  31: (state) => ({
+    ...state,
+    quranReaderStyles: {
+      ...state.quranReaderStyles,
+      ...(state.quranFont === QuranFont.Tajweed && {
```

Comment:
`state.quranFont` is not a valid value, it should be `state.quranReaderStyles.quranFont`

Reply:
I was wondering why it is no longer working xD